crates/runtime/tests/graphql/mod.rs

```diff
@@ -304,7 +304,7 @@ async fn test_graphql_pagination() -> Result<(), String> {
             &format!("http://{addr}/graphql"),
             "test_graphql",
             "query { paginatedUsers(first: 2) { users { id name posts { id title content } } pageInfo { hasNextPage endCursor } } }",
-            "data.paginatedUsers.users",
+            "/data/paginatedUsers/users",
```

Comment:
do we have some escape rule for json pointer?

Reply:
a quick ai search says the json pointer and json path use different escape rule, which might be worthy to check, and potential add some tests if that is considered important.

Reply:
The code actually previously just converted the path to a json pointer.  I'll double check the escaping.

Reply:
We used Value.pointer() from serde_json Value.  It does all of the escaping and conforms to the RFC.  Since we're not adding additional logic here it doesn't make senses to add testing around this.  

https://docs.rs/serde_json/latest/serde_json/enum.Value.html